client/extensions/woocommerce/app/order/create.js

```diff
@@ -50,13 +52,33 @@ class Order extends Component {
 		this.props.clearOrderEdits( this.props.siteId );
 	}
 
+	triggerInvoice = ( siteId, orderId ) => {
+		const { translate } = this.props;
+		const onSuccess = dispatch => {
+			dispatch(
+				successNotice( translate( 'An invoice has been sent to the customer.' ), {
+					duration: 5000,
```

Comment:
Product and promotion creation have durations of 8000. Maybe we should increase this to match? 5000 feels super fast to me :)